### PR TITLE
Include `env` as a dependency for `up` just recipe

### LINKED ISSUE #964: Quickstart guide doesn't mention `just env`
The quickstart guide as written will error upon the first `just up` because there's no `.env` file.

We can either ask users to `just env`, or we can consider including it elsewhere as part of `just install`, or again some alternative approach.

### PR DESCRIPTION
**Pull request by AetherUnbound**
## Fixes

Fixes #964 by @zackkrida

## Description

~~This PR adds the `env` just recipe as a dependency of both the `py-install` and `node-install` recipes, then changes the syntax for the `install` recipe to use just's implicit dependency management. This ensures `env` is run once rather than for each install command when `just install` is run.~~ 

~~Since `up` is the more common recipe, adding it as a dependency to `install` seems reasonable since it shouldn't be necessary to run that frequently.~~

Per @sarayourfriend's comment below, I have change this so `env` is now a dependency of `up`.

## Testing Instructions

I tested this by cloning a fresh repo (not a quick process!) and ensuring that the steps in the Quickstart guide (i.e. `just install` -> `just up`) did indeed succeed now because the `.env` files are created.

Running `just install` should be sufficient for testing though, as that will show the env steps running.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by sarayourfriend — COMMENTED** [review]
I don't really understand why not adding it to `up`. `just env` takes less than a tenth of a second to run (maybe less than a 20th of a second) and, as it is the more common one to run, will catch _every_ case where it is necessary to have those .env files rather than just the initial installation (which doesn't directly depend on it and is itself not even necessary for just running the application locally without development tools).

```
➜  openverse git:(fix/region-name-template-rendering) ✗ time just env
# Root
([ ! -f .env ] && cp env.template .env) || true
# Docker
([ ! -f docker/minio/.env ] && cp docker/minio/env.template docker/minio/.env) || true
# First-party services
([ ! -f catalog/.env ] && cp catalog/env.template catalog/.env) || true
([ ! -f ingestion_server/.env ] && cp ingestion_server/env.template ingestion_server/.env) || true
([ ! -f api/.env ] && cp api/env.template api/.env) || true
just env  0.07s user 0.03s system 122% cpu 0.084 total
```

`env` should be made into a silent recipe though, it certainly doesn't need to print all of that every time _anything_ is run.

File: justfile [hunk 1]
```diff
@@ -45,20 +45,18 @@ ENABLE_DC_OVERRIDES := env_var_or_default("OPENVERSE_ENABLE_DC_OVERRIDES", "true
...
-py-install:
+py-install: env
     just automations/python/install
     just documentation/install
 
 # Install all dependencies
-install:
-    just node-install
-    just py-install
+install: node-install py-install
```
Comment (dhruvkb) [inline justfile:59]: This is not aligned with the general convention in our `justfile`s. Alternatively we could adopt this and then there's a lot more instances of recipes just being a collection of other `just` recipe calls.

**Review by dhruvkb — APPROVED** [review]
LGTM, except a minor nit.

**Comment by AetherUnbound** [comment]
I have no strong feelings for which recipe it should be attached to. I'll revert the changes to the `install` recipe and `env` a dependency of `up` instead.

**Review by sarayourfriend — APPROVED** [review]
LGTM :+1:

**Pull request merged**